crates/hc/src/lib.rs

```diff
@@ -74,19 +90,19 @@ pub enum Opt {
     /// Work with sandboxed environments for testing and development.
     Sandbox(hc_sandbox::HcSandbox),
     /// Allow redirect of external subcommands (like `hc-scaffold` and `hc-launch`).
-    #[structopt(external_subcommand)]
+    #[command(external_subcommand)]
     External(Vec<String>),
 }
 
-impl Opt {
+impl Cli {
     /// Run this command.
     pub async fn run(self) -> anyhow::Result<()> {
-        match self {
-            Self::Dna(cmd) => cmd.run().await?,
-            Self::App(cmd) => cmd.run().await?,
-            Self::WebApp(cmd) => cmd.run().await?,
-            Self::Sandbox(cmd) => cmd.run().await?,
-            Self::External(args) => {
+        match self.command {
+            CliCommand::App(cmd) => cmd.command.run().await?,
+            CliCommand::Dna(cmd) => cmd.command.run().await?,
+            CliCommand::WebApp(cmd) => cmd.command.run().await?,
```

Comment:
Could the `impl` be moved down a level to the inner enum rather than the struct so this stays as `cmd.run().await`? To be consistent with the sandbox command on the next line

Reply:
sure, that'd simplify things. I'll do that today and push changes.

I'm learning so much about Rust!

Reply:
I was agreeing with it being nice for this to just be `cmd.run()`, which I guess would actually require an impl on `cmd` in each of these cases. Maybe the better way to make it consistent is to remove that impl on HcSandbox so it's `cmd.command.run()` there too! Either way it's very minor, I'm ok with it as is.

Reply:
I opted to put `run` on the weird wrapper structs I had to create for compatibility with the newest clap. The only exception is the three bundling subcommands, because recursive packing calls `HcDnaBundleSubcommand::Pack.run()`, and it seemed silly to create a `HcDnaBundle` instance just to stuff the subcommand in there and call `bundle.run()` on it. So `HcDnaBundle::run()` exists for consistency's sake, but it just calls `self::subcommand.run()`.

Incidentally, I also renamed `*Command` to `*Subcommand`, cuz it seemed more intuitive.